<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav1)"/>
  <arg name="debug" default="false" />
  <arg name="print_debug" default="true" />

  <arg unless="$(arg debug)" name="launch_prefix" value=""/>
  <arg     if="$(arg debug)" name="launch_prefix" value="debug_roslaunch"/>

  <group if="$(arg print_debug)">
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find balloon_planner)/config/rosconsole/debug.conf"/>
</group>
  <group ns="$(arg uav_name)">

    <node pkg="nodelet" type="nodelet" name="balloon_planner" args="standalone balloon_planner/BalloonPlanner" output="screen" launch-prefix="$(arg launch_prefix)" >

      <param name="uav_name" type="string" value="$(arg uav_name)" />
      <param name="uav_frame_id" type="string" value="fcu_$(arg uav_name)" />
      <rosparam file="$(find balloon_planner)/config/sim_plan_params.yaml" command="load" />

      <!-- Subscribers -->
      <remap from="~balloon_detections" to="object_detect/detected_objects_" />

      <!-- Publishers -->
      <remap from="~balloon_chosen_out" to="~balloon_chosen_out_" />

    </node>

  </group>

</launch>
